### PR TITLE
readme: fix a link

### PR DESCRIPTION
**Pull request by dcalvin**
Fixed a link in readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Your journey of becoming a contributor and committer starts from here: improving
 
 ## Communicating
 
-You can reach out to the community members via any one of the following ways: 
+You can reach out to the community members via any one of the following ways:
 
 + Slack: [https://tikv-wg.slack.com](https://tikv-wg.slack.com)
```
Comment (gingerkidney) [inline README.md]: this is not an invite link. you clould replace it with https://join.slack.com/t/tikv-wg/shared_invite/enQtNTUyODE4ODU2MzI0LWVlMWMzMDkyNWE5ZjY1ODAzMWUwZGVhNGNhYTc3MzJhYWE0Y2FjYjliYzY1OWJlYTc4OWVjZWM1NDkwN2QxNDE

Comment (dcalvin) [inline README.md]:
```suggestion
+ Slack: [https://tikv-wg.slack.com](https://join.slack.com/t/tikv-wg/shared_invite/enQtNTUyODE4ODU2MzI0LWVlMWMzMDkyNWE5ZjY1ODAzMWUwZGVhNGNhYTc3MzJhYWE0Y2FjYjliYzY1OWJlYTc4OWVjZWM1NDkwN2QxNDE)
```

**Comment by dcalvin** [comment]
PTAL again @gingerkidney

**Comment by gingerkidney** [comment]
LGTM

**Pull request merged**